app/controllers/AccountManagement.scala

```diff
@@ -297,17 +298,15 @@ object AccountManagement extends Controller with LazyLogging with CatalogProvide
     implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
```

Comment:
this sub.currentPlan.charges.currencies.head is pretty messy and unnecessary (I know it's existing) maybe we can add something to subscriptionOps?